extensions/panache/hibernate-orm-panache-common/runtime/src/main/java/io/quarkus/hibernate/orm/panache/common/runtime/CommonPanacheQueryImpl.java

```diff
@@ -93,6 +109,12 @@ private CommonPanacheQueryImpl(CommonPanacheQueryImpl<?> previousQuery, String n
         return new CommonPanacheQueryImpl<>(this, select.toString() + query, "select count(*) " + query);
     }
 
+    public void filter(String filterName, Map<String, Object> parameters) {
+        if (filters == null)
+            filters = new HashMap<>();
```

Comment:
We didn't lazily instantiate the hint Map.
If you think it worth it, can you refactor the hint Map to be lazily instantiated for coherence sake ?

Reply:
It's something I like to do for <10% use-cases just in case. You can do it for the other stuff if you want or wait until a later refactor. No rush.

Reply:
Oh, sure, I can add it here.

Reply:
Done.